aws-ssm-ec2-proxy-command.sh

```diff
@@ -46,13 +46,13 @@ then
   ec2_instance_id="${ec2_instance_id%%${REGION_SEPARATOR}*}"
 fi
 
->/dev/stderr echo "Add public key ${ssh_public_key_path} to instance ${ec2_instance_id} for 60 seconds"
+>/dev/stderr echo "Add public key ${ssh_public_key} at path ${ssh_public_key_path} to instance ${ec2_instance_id} as ${ssh_user} for ${ssh_public_key_timeout} seconds"
```

Comment:
Could you remove `${ssh_public_key} at path` it is a little too verbose in my opinion.